Week-3/Media queries for a vacation website/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Media Query Lab</title>
+    <link rel="stylesheet" href="styles.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <header>
+      <h1>Mediterranian Vacations</h1>
+    </header>
+    <nav>
+      <ul>
+        <li><a href="#featured">Featured Packages</a></li>
+        <li><a href="#exotic">Exotic Vacations</a></li>
+        <li><a href="#historic">Historic Locations</a></li>
+      </ul>
+    </nav>
+    <div class="benefits-container">
+      <div class="benefits-inner-container">
+        <div>
+          <img src="checkmark.png" alt="checkmark" />
```

Comment:
always try to use relative path this is absolute path, these type of things will not work in multi env